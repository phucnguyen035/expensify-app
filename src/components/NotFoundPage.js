import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div className="not-found">
    <img className="not-found__image" src="/images/sad.png" alt="sad face" />
    <p className="not-found__text">Uh oh, the page you were looking for does not exist.</p>

    <Link className="not-found__link" to="/">
      Go to home
    </Link>
  </div>
);

export default NotFoundPage;
