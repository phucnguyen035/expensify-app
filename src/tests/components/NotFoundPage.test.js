import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

describe('404 Page', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<NotFoundPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
