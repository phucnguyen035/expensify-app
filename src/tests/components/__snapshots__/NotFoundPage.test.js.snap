// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page should render correctly 1`] = `
<div
  className="not-found"
>
  <img
    alt="sad face"
    className="not-found__image"
    src="/images/sad.png"
  />
  <p
    className="not-found__text"
  >
    Uh oh, the page you were looking for does not exist.
  </p>
  <Link
    className="not-found__link"
    replace={false}
    to="/"
  >
    Go to home
  </Link>
</div>
`;
